src/material/list/list.spec.ts

```diff
@@ -549,19 +548,6 @@ class ListWithTwoLineItem extends BaseTestList {}
 })
 class ListWithThreeLineItem extends BaseTestList {}
 
-@Component({
-  template: `
-  <mat-list>
-    <mat-list-item *ngFor="let item of items">
-      <h3 matListItemTitle>Line 1</h3>
-      <p matListItemLine>Line 2</p>
-      <p matListItemLine>Line 3</p>
-      <p matListItemLine>Line 4</p>
-    </mat-list-item>
-  </mat-list>`,
-})
-class ListWithManyLines extends BaseTestList {}
```

Comment:
Shoild we have a test for the sanity check?

Reply:
Potentially, but I was thinking that we may have to turn it into a `console.warn`. It broke some internal targets which means that it'll break externally as well.

Reply:
Fair enough. In either case, would be good to verify that the sanity checks work properly I think

Reply:
Not in particular. I believe it just ended up being a warning anyway, using the error logging